Automattic/wp-calypso
# Allow a plan name to be displayed properly on the post-purchase authorization screen

## Proposed Changes

* This PR updates the post-connection authorization screen for Jetpack to correct a bug when a plan (not an individual product) was purchaed.

## Testing Instructions
* Start with a new Jetpack site
* From My Jetpack, while disconnected, click "Learn More" on the backup card and click to "Get Security"
* Without this patch, try completing the purchase - you should get an error screen just after checkout
* With this patch applied locally, copy and paste the checkout URL and try the checkout again in your local environment. It should complete the purchase and allow you to connect Jetpack afterwards

## Pre-merge Checklist
